lib/perl/Genome/InstrumentData/Composite/Strategy.pm

```diff
@@ -90,6 +92,9 @@ sub grammar {
         | "aligned" "to" reference "using" aligner
             { $return = $item[5]; $item[5]->{reference} = $item[3]; $item[5]->{type} = 'align'; }
 
+        align_and_merge: "align" "and" "merge" "to" reference "using" aligner
```

Comment:
All of the other actions in the alignment strategies are `-ed` verbs.  One option might be: `"both" "aligned" "to" reference "and" "merged" "using" aligner`.